pages/vision.tsx

```diff
@@ -0,0 +1,92 @@
+import Head from "next/head";
+import Table from "./components/Table/tables";
+import { Inter } from "@next/font/google";
+import styles from "../styles/Home.module.scss";
+import Button from "./components/common/Button/Button";
+import Navbar from "./components/Navbar/Navbar";
+import Card from "./components/common/Card/Card";
+import AccordionGrid from "./components/AccordionGrid/AccordionGrid";
+import Footer from "./components/Footer/Footer";
+import Link from "next/link";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export default function Home() {
+  return (
+    <>
+      <Head>
+        <title>Wesh Network</title>
+        <meta name="description" content="Wesh Network" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.Body}>
+        <Navbar />
+        <section className={styles.Section6}>
+          <div className={styles.left}>
+            <div className={styles.Section6Text}>
+              <h2 className={styles.green}>VISION</h2>
+            </div>
+            <div className={styles.Section6Text}>
+              <h3 className={styles.space}>The cornerstone of the internet of tomorrow</h3>
+            </div>
+            <div className={styles.Section6Grid}>
+              <p className={styles.space}>Our ambition is to establish a decentralized communication protocol as a cornerstone of the internet of tomorrow. We aim to empower web3 developers to bring about unstoppable peer-to-peer communication to billions of devices. </p>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>We are driven by a mission to empower humanity with the power of unrestricted communication, free from the limitations of centralized infrastructure and censorship.</p>
+            </div>
+            <div className={styles.Section6Grid}>
+
+              <p className={styles.space}>Our open-source approach promotes contributions and transparency, allowing anyone to take part in our pursuit of uncompromised communication.</p>
+            </div>
+
+          </div>
+          <div className={styles.right}>
```

Comment:
In my browser, there are many centimeters between the previous section and the "Crossing the Chasm" section. Maybe it is caused by this image? I think there should be less space. On some computer screens, it may not be clear that they need to scroll far down to read more.